base/applications/network/arp/arp_msg.mc

```diff
@@ -81,6 +82,37 @@ Przykłady:
   > arp -s 157.55.85.212   00-aa-00-62-c6-09  .... Dodaje statyczny wpis.
   > arp -a                                    .... Wyświetla tabelę arp.
 .
+Language=Romanian
+Afișează și modifică tabelele de traducere din adrese IP în adrese fizice folosite de
+Protocol de rezolvare a adreselor (ARP).
+
+ARP -s inet_addr eth_addr [if_addr]
+ARP -d inet_addr [if_addr]
+ARP -a [inet_addr] [-N if_addr]
+
+  -a            Afișează intrările ARP actuale interogând datele protocolului
+                actual. Dacă inet_addr e specificat, adresele IP și fizică
+                sunt afișate numai pentru calculatorul specificat. Dacă
+                mai mult de o interfață de rețea folosește ARP, sunt afișate intrările
+                pentru fiecare tabel ARP.
+  -g            La fel ca la -a.
+  inet_addr     Specifică o adresă de internet.
+  -N if_addr    Afișează intrările ARP pentru interfața de rețea specificată
+                de if_addr.
+  -d            Șterge gazda specificată de inet_addr. inet_addr poate fi
+                marcat cu caracterul special * pentru a șterge toate gazdele.
+  -s            Adaugă gazda și asociază adresa de Internet inet_addr
+                cu adresa fizică eth_addr. Adresa fizică este
+                reprezentată ca 6 octeți hexazecimali separați de cratime. Intrarea
+                este permanentă.
+  eth_addr      Specifică adresa fizică.
+  if_addr       Dacă e prezentă, aceasta specifică adresa de Internet a
+                interfeței al cărui tabel de traducere a adreselor trebuie modificat.
```

Comment:
I am not clear about your "al" article. With which word it is supposed to agree (in gender)?

Reply:
With the word "tabel" which is neutral.

Reply:
I revised the sentence. However, I don't know if it can be made shorter.

Reply:
> I revised the sentence. However, I don't know if it can be made shorter.

You can always complete a shorter line with words cut from the next line below it, in order to get closer to the 80 characters limit. The same way, a longer line can be cut shorter by cutting it at the end and pasting those at the beginning of the next line's contents or into a new line (if there isn't one).
For example, the line starting with "interfeței" (after an indent of several spaces) breakes like this:
`                interfeței al cărei tabel de traducere a adreselor ar trebui`
`                modificat.`

Reply:
Good, thank you! I did it.